docs/detections/rules-ui-manage.asciidoc

```diff
@@ -101,7 +104,8 @@ NOTE: You cannot export Elastic prebuilt rules.
 .. Drag and drop the file that contains the detection rules.
 +
 NOTE: Imported rules must be in an `.ndjson` file.
-.. Select *Overwrite existing detection rules with conflicting Rule ID* to update existing rules if they match any rules in the import file. Configuration data included with the rules, such as actions, are also overwritten.
+.. Select *Overwrite existing detection rules with conflicting "rule_id"* to update existing rules if they match the `rule_id` value of any rules in the import file. Configuration data included with the rules, such as actions, is also overwritten.
+.. Select *Overwrite existing exception lists with conflicting "list_id"* to replace existing exception lists with exception lists from the import file if they have a matching `list_id` value.
```

Comment:
Are these overwriting actions optional? Seems like they might be, but it also feels as though we're encouraging users to select these options--sort of like a best practice when importing rules.

Reply:
Yes, these are optional check boxes. I think adding "(Optional)" to each step is an elegant solution.